scripts/limonade/src/index.ts

```diff
@@ -138,6 +177,36 @@ async function run() {
     }
   };
 
+  const isOffMarket = (order: { state: Order; address: PublicKey }) => {
+    const priceInputMint = priceStore[order.state.inputMint.toString()];
+    const priceOutputMint = priceStore[order.state.outputMint.toString()];
+
+    if (!priceInputMint || !priceOutputMint) {
+      // If we don't have price info, we will not consider it off-market
+      return false;
+    } else {
+      const inputAmount = order.state.remainingInputAmount;
```

Comment:
inputAmount can be `initialInputAmount` and outputAmount can be expectedOutputAmount without the subtraction.
This way, we don't need to assume the inputAmount is positive

Reply:
I like it this way since it reflects more the current state of the order. 
Besides just before filtering by `isWithinMarketPriceBand` with filter by `remainingInputAmount != 0` so we know we won't divide by 0 here.